src/extractor/scripting_environment_lua.cpp

```diff
@@ -276,6 +288,71 @@ void Sol2ScriptingEnvironment::InitContext(LuaScriptingContext &context)
         "version",
         &osmium::Way::version);
 
+    struct RelationMemberWrap
+    {
+        explicit RelationMemberWrap(const osmium::RelationMember & member)
+        {
+            init(member);
+        }
+
+        RelationMemberWrap()
+            : item_type(osmium::item_type::undefined)
+        {
+        }
+
+        void init(const osmium::RelationMember & member)
+        {
+            role = member.role();
+            item_type = member.type();
+            id = member.ref();
+        }
+
+        const char* GetRole() const { return role.c_str(); }
```

Comment:
you don't need accessor function with pubic data members  , it is better to pass references to sol and set r/o access as `sol::readonly( &RelationMemberWrap::role )`

Reply:
done